### DISCUSSION TITLE
Why does Insiders start at $15/month?

### DISCUSSION THREAD

**Discussion by AjaniBilby** [squidfunk/mkdocs-material#4524] [Q&A] [answered]
> To be clear I'm not asking for pricing to change at all, I'm just wanting to understand how the tiers were chosen

I'm wondering where exactly the $15/mo individual rate for the insiders entry tier came from?  
It feels like an odd price considering Github pro is $4/mo, and Youtube Premium is $12/mo as a point of comparison.  
If I was a full time developer and updated my website multiple times a month, I probably wouldn't blink at the $15 price tag, but as a student that seems like a high price for the value I'm getting.

To be clear, I'm not saying you should lower prices or add a student tier, because the admin time alone wouldn't be worth it.

**Comment by squidfunk** [comment, marked as answer]
Excellent question, thanks for asking! First of all, I agree that it's awesome that GitHub is so cheap at $4/month, but it's also a multi-billion dollar company that can afford to offer basic accounts at this price, and then monetize with added services like build minutes on GitHub Actions etc. You can probably imagine, that as an independent developer, it's much, much harder to build a living. I just cannot afford to offer prices as low as this, it's just not economical for me.

Furthermore, I'm giving you __everything I have for $15/month__ + updates, bugfixes, try to answer all of your questions, help you achieve what you want, etc. If you pay $15/month for a year (= $180), it's still less than what I bill companies by the hour, offering them 17 years of experience building web applications from the ground up. I'm very happy about all organizations joining the program at $125/month, which buys me more time to help the entire community. It's a mixed calculation after all.

I hope you understand my reasoning. I think it's safe to say that nobody wants to work for $4/month (= $48/year) 😉

**Reply by AjaniBilby** [reply]
I guess that's a different way of thinking about it which totally makes sense. I was thinking about pricing in terms of X cost divided between Y users, which is somewhat closer to probably how the pricing was determined for the examples I listed. Where as of course you could think about it in terms of how many hours did you save me if I did this myself, then how much would that cost to hire someone to do that. In which case the value is more than there.

**Reply by squidfunk** [reply]
It boils down to the question if you want to build and maintain the features that are in Insiders yourself for $15/month (or more for organizations). I believe that this is a fair deal, and I commit to maintain the project and not abandon it at some point.
